clustered/client/src/test/java/org/ehcache/clustered/client/CacheManagerDestroyTest.java

```diff
@@ -124,14 +125,15 @@ public void testDestroyCacheManagerDoesNotAffectsExistingCacheWithExistingClient
 
     assertThat(cache.get(1L), is("One"));
 
+    persistentCacheManager2.close();
```

Comment:
I have been wondering that ... but then I'd rather have it handled by passthrough server. Something like `destroyAllAndStop()`. Makes sense?